index.html

```diff
@@ -3,61 +3,52 @@
 
 <head>
   <meta charset="utf-8" />
-  <title></title>
-  <meta name="author" content="" />
-  <meta name="description" content="" />
+  <title>My daily work</title>
+  <meta name="author" content="flavia spataru" />
+  <meta name="description" content="user needs" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <!-- <link href="css/normalize.css" rel="stylesheet"> -->
-  <link href="style.css" rel="stylesheet" />
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
     integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous" />
+  <link href="style.css" rel="stylesheet" />
 </head>
 
 <body>
-  <main>
+  <header>
     <nav>
       <a href="#tasks" class="tab-item active" id="tasks">Tasks</a>
       <a href="#faq" class="tab-item active" id="faq">FAQ</a>
       <a href="#contact" class="tab-item active" id="contact">Contact us</a>
-      <!-- <a href="javascript:void(0);" class="icon" onclick="myFunction()">
-          <i class="fa fa-bars"></i>
-        </a> -->
     </nav>
+  </header>
+  <main>
     <div class="tab-content-item show" id="tasks-content">
-      This is for Tasks
-      <div class="row">
-        <div class="col-4">
-          <div class="list-group" id="list-tab" role="tablist">
```

Comment:
Don't overcrowd the DOM with unnecessary elements ([https://web.dev/dom-size/]) - spre ex, div-ul de aici nu e necesar, poti muta clasele si id-ul pe div-ul de mai sus